src/development/platform-integration/ios/c-interop.md

```diff
@@ -241,42 +241,6 @@ use the following instructions:
 1. Also add it to the **Linked Frameworks & Libraries**
    section of the target in Xcode.
 
-#### Compiled (dynamic) library (macOS)
```

Comment:
I removed this section since its duplicated on the macOS c-interop page which seems where it's targeted to.